postgres-driver/block.go

```diff
@@ -70,17 +71,20 @@ func (d *PostgresDriver) WriteBlock(block *indexer.Block) error {
 type ReadBlocksOptions struct {
 	PerPage int
 	Page    int
+	Order   provider.Order
```

Comment:
I think this enum should also be defined in this repo. I don't see any advantage of adding the provider dependency just for that.